.circleci/pr-halt-e2e.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Script name: should this start with `e2e` for consistency?

Reply:
renamed to `skip-e2e-pr-checks.sh` (verb first)